packages/mapped-reducer/readme.md

```diff
@@ -125,7 +125,7 @@ Get a subreducer for the given action type(s).
 
 #### `#reduce(state: State, action: Action): State`
 
-This is the actual reducing method. You should pass this to `createStore` or `combineReducers`.
+This is the actual reducing method. You should pass this to `createStore` or `combinedReducers`.
```

Comment:
`combineReducers ` is not a mistake. It is a function from Redux.
I'll fix this.

Reply:
ah ok. I thought you were referencing a variable from the code snippet